KegHub/keg-cli
# Docer build platform

## Context

* Why was this ticket created?

## Updates

* List the updates you made to the codebase. Use your best judgement on how detailed the description should be
* *Name of file modified*
  * What changed in this file? Why?

## Testing

* List out the steps you took to test your changes
* Use the template below for listing out your testing steps
  ```
    * <Function> Testing:
      * Step 1
      * Step 2
      * Step n
      * Verify <some_expectations>
  ```
* (Example) Search Testing:
  * login to zsales.zerista.com as admin
  * Click on the 'Speakers' tab
  * Search for 'billy bob'
  * Verify that the speaker user shows up
